WTWSMS/decisions/realm_decisions.txt

```diff
@@ -1271,13 +1271,16 @@ decisions = {
 		potential = {
 			has_dlc = "Sons of Abraham"
 			is_playable = yes
-			religion_group = jewish_group
+			OR = {
+				religion_group = jewish_group
+				religion = ebionites
+			}
 			k_israel = {
 				has_holder = no
 			}
 		}
 		allow = {
-			completely_controls = k_oriens
+			completely_controls = d_iudea # K_ORIENS IS TOO BIG, THIS IS WHAT ISRAEL HAD IN RL
```

Comment:
Respectively:

1.  The original division of Caanan by the Jews, split among the Twelve Tribes of Israel.
2. Israel at its peak, under King Solomon.

Notice their near identity with D_IUDEA . There is a reason that duchy is as it is.